chat-plugins/uno.js

```diff
@@ -0,0 +1,703 @@
+/*
+	UNO Game for PS
+
+	by sparkychild
+*/
+'use strict';
+const deck = ["R1",
+	"R2",
+	"R3",
+	"R4",
+	"R5",
+	"R6",
+	"R7",
+	"R8",
+	"R9",
+	"RS",
+	"RR",
+	"R+2",
+	"Z+4",
+	"Y1",
+	"Y2",
+	"Y3",
+	"Y4",
+	"Y5",
+	"Y6",
+	"Y7",
+	"Y8",
+	"Y9",
+	"YS",
+	"YR",
+	"Y+2",
+	"Z+4",
+	"G1",
+	"G2",
+	"G3",
+	"G4",
+	"G5",
+	"G6",
+	"G7",
+	"G8",
+	"G9",
+	"GS",
+	"GR",
+	"G+2",
+	"Z+4",
+	"B1",
+	"B2",
+	"B3",
+	"B4",
+	"B5",
+	"B6",
+	"B7",
+	"B8",
+	"B9",
+	"BS",
+	"BR",
+	"B+2",
+	"Z+4",
+	"R1",
+	"R2",
+	"R3",
+	"R4",
+	"R5",
+	"R6",
+	"R7",
+	"R8",
+	"R9",
+	"RS",
+	"RR",
+	"R+2",
+	"ZW",
+	"R0",
+	"Y1",
+	"Y2",
+	"Y3",
+	"Y4",
+	"Y5",
+	"Y6",
+	"Y7",
+	"Y8",
+	"Y9",
+	"YS",
+	"YR",
+	"Y+2",
+	"ZW",
+	"Y0",
+	"G1",
+	"G2",
+	"G3",
+	"G4",
+	"G5",
+	"G6",
+	"G7",
+	"G8",
+	"G9",
+	"GS",
+	"GR",
+	"G+2",
+	"ZW",
+	"G0",
+	"B1",
+	"B2",
+	"B3",
+	"B4",
+	"B5",
+	"B6",
+	"B7",
+	"B8",
+	"B9",
+	"BS",
+	"BR",
+	"B+2",
+	"ZW",
+	"B0",
+];
+//declare html for DRAW and PASS button.
+const drawButton = '<center><button style="background: black; border: 2px solid rgba(33 , 68 , 72 , 0.59) ; width: 56px ; border-radius: 5px , auto" name="send" value="/uno draw"><font color="white">Draw</font></button></center>';
+const passButton = '<center><button style="background: red; border: 2px solid rgba(33 , 68 , 72 , 0.59) ; width: 56px ; border-radius: 5px , auto" name="send" value="/uno pass"><font color="white">PASS</font></button></center>';
+
+//used for the Reverse card
+function invertArray(array) {
```

Comment:
ok i'll fix that; sorry about the npm test thing; c9's npm test is a lot more lenient than github's
Thanks for the tip about Array.reverse()